aiida_pseudo/groups/mixins/cutoffs.py

```diff
@@ -85,13 +85,13 @@ def validate_stringency(self, stringency: str) -> None:
         """Validate a cutoff stringency.
 
         :param stringency: the cutoff stringency to validate.
-        :raises ValueError: if stringency is None or the family does not define cutoffs for the specified stringency.
+        :raises ValueError: if the family does not define cutoffs for the specified stringency.
         """
-        if stringency is None:
-            raise ValueError('defining a stringency is required.')
-
         if stringency not in self.get_cutoff_stringencies():
-            raise ValueError(f'stringency `{stringency}` is not defined for this family.')
+            raise ValueError(
+                f'stringency \'{stringency}\' is not one of the available cutoff stringencies for this family: '
```

Comment:
```suggestion
                f'stringency `{stringency}` is not one of the available cutoff stringencies for this family: '
```

Reply:
I know we've been using backticks for this purpose a lot, but is there any reason to prefer backticks here over single quotes, since the stringency is actually a string?

Reply:
Just for consistency. I don't think we need the literal single quotes to make it clear this is a string value and just having backticks everywhere to reflect values that were "dynamic" and should interest the caller specifically, seems useful to me. Also, backticks don't have to be escaped :grimacing:

Reply:
True, it's cleaner and plenty clear. Just decided to try it and see what happens. 😁